orc/src/main/java/org/apache/iceberg/orc/OrcRowWriter.java

```diff
@@ -35,4 +37,12 @@
    * @throws IOException if there's any IO error while writing the data value.
    */
   void write(T row, VectorizedRowBatch output) throws IOException;
+
+  /**
+   * Returns a stream of {@link FieldMetrics} that this OrcRowWriter keeps track of.
+   * <p>
+   * Since ORC keeps track of most metrics via column statistics, for now OrcRowWriter only keeps track of NaN
+   * counters for double or float columns.
```

Comment:
I don't think this paragraph needs to be here because it is a snapshot of how another component works. It could get stale really easily.

Reply:
Makes sense, I'll remove similar instances in here, `OrcValueWriter` and `ParquetValueWriter`